client/client.go

```diff
@@ -120,8 +128,14 @@ func New(config *Config) *Client {
 		panic(fmt.Sprintf("cannot parse server base URL: %q (%v)", config.BaseURL, err))
 	}
 	return &Client{
-		baseURL:     *baseURL,
-		doer:        &http.Client{Transport: &http.Transport{DisableKeepAlives: config.DisableKeepAlive}},
+		baseURL: *baseURL,
+		doer: &http.Client{Transport: &httputil.LoggedTransport{
+			Transport: &http.Transport{
+				DisableKeepAlives: config.DisableKeepAlive,
+			},
+			Key:  "SNAP_CLIENT_DEBUG_HTTP",
+			Body: true,
```

Comment:
do we need to do somethin special about InstallPath ? I notice that the store package code sets up a client where this is false when doing downloads for example